examples/fleet/kuard/README.md

```diff
@@ -0,0 +1,184 @@
+# Example: Exporting service using Fleet resource placement
+
+This directory contains the instructions on how to export services using Fleet resource placement and setup Layer 4 load balancing across workloads deployed across fleet member clusters.
+
+The application is using [kuard](https://github.com/kubernetes-up-and-running/kuard) as the service to demonstrate the responding cluster/pod.
+
+## Before you begin
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/).
+* Create a fleet with three members: `aks-member-1`, `aks-member-2`, and `aks-member-3` using [Azure CNI networking](https://review.learn.microsoft.com/en-us/azure/aks/configure-azure-cni).
+* Member clusters reside on the same virtual network, or [peered virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview) such that pods from different clusters can communicate directly with each other using pod IPs.
+* Configure kubectl to communicate with your hub cluster.

```

Comment:
is there a link to how to do this or we assumed that users have done that.

Reply:
it will be good to name the `kubeconfig` file as `hub` so we can use it below

Reply:
we don't have the public doc yet, added command in the doc